dace/transformation/dataflow/prune_connectors.py

```diff
@@ -116,7 +116,7 @@ def apply(self, state: SDFGState, sdfg: SDFG):
                         break
 
 
-class PruneSymbols(pm.SingleStateTransformation, pm.SimplifyPass):
+class PruneSymbols(pm.SingleStateTransformation):
```

Comment:
Was it found that PruneSymbols removes symbols that it shouldn't?

Reply:
It is correct, but slow for large SDFGs. In the next three PRs (Passes) it will become faster since it won't be applied on a pattern-matching basis.